container_opts_unix.go

```diff
@@ -61,30 +68,39 @@ func withRemappedSnapshotBase(id string, i Image, uid, gid uint32, readonly bool
 		if err != nil {
 			return err
 		}
+		if remapper == "fuse-overlayfs" {
```

Comment:
I think `remapper` value should be `snapshotter`.
Other snapshotter implementations may support the same mapping labels.

Reply:
also we should typedef remapper.

Reply:
Do you mean the `snapshotter` from https://github.com/containerd/containerd/blob/master/snapshots/snapshotter.go#L241
for the `remapper`?

Reply:
So remove the `client.resolveSnapshotterName(ctx, c.Snapshotter)` code and pass in a `Snapshotter`?

Reply:
just like this

```go
type Remapper string
const (
  remapperChown Remapper = "chown"
  remapperSnapshotter Remapper = "snapshotter"
)
```

Reply:
Added typedef for remapper.